projects/plugins/jetpack/extensions/plugins/publicize/settings.js

```diff
@@ -1,60 +1,25 @@
 import {
-	PostPublishManualSharing,
-	PostPublishReviewPrompt,
 	PublicizePanel,
 	SocialImageGeneratorPanel,
-	useSyncPostDataToStore,
 	usePublicizeConfig,
-	useSocialMediaConnections,
 } from '@automattic/jetpack-publicize-components';
-import { JetpackEditorPanelLogo } from '@automattic/jetpack-shared-extension-utils';
-import { PluginPrePublishPanel } from '@wordpress/edit-post';
-import { PostTypeSupportCheck } from '@wordpress/editor';
-import { __ } from '@wordpress/i18n';
-import JetpackPluginSidebar from '../../shared/jetpack-plugin-sidebar';
 import UpsellNotice from './components/upsell';
+import PostPublishPanels from './post-publish';
+import PrePublishPanels from './pre-publish';
 
-export const Settings = () => {
-	useSyncPostDataToStore();
-	const { hasEnabledConnections } = useSocialMediaConnections();
+const Settings = () => {
 	const { isSocialImageGeneratorAvailable } = usePublicizeConfig();
 
 	return (
-		<PostTypeSupportCheck supportKeys="publicize">
-			<JetpackPluginSidebar>
-				<PublicizePanel>
-					<UpsellNotice />
-				</PublicizePanel>
-				{ isSocialImageGeneratorAvailable && <SocialImageGeneratorPanel /> }
-			</JetpackPluginSidebar>
-
-			<PluginPrePublishPanel
```

Comment:
I think it might be this. With the refactor, the PluginPrePublishPanel is being rendered inside the JetpackPluginSidebar component.